diff -ur a/ext/opcache/zend_accelerator_util_funcs.c b/ext/opcache/zend_accelerator_util_funcs.c
--- a/ext/opcache/zend_accelerator_util_funcs.c	2016-08-17 15:51:56.000000000 -0400
+++ b/ext/opcache/zend_accelerator_util_funcs.c	2016-09-03 17:29:48.644478053 -0400
@@ -648,7 +648,7 @@
 	return;
 }
 
-#ifdef __SSE2__
+#if 0
 #include <mmintrin.h>
 #include <emmintrin.h>
 
@@ -689,7 +689,7 @@
 		ZCG(current_persistent_script) = persistent_script;
 		ZCG(arena_mem) = NULL;
 		if (EXPECTED(persistent_script->arena_size)) {
-#ifdef __SSE2__
+#if 0
 			/* Target address must be aligned to 64-byte boundary */
 			ZCG(arena_mem) = zend_arena_alloc(&CG(arena), persistent_script->arena_size + 64);
 			ZCG(arena_mem) = (void*)(((zend_uintptr_t)ZCG(arena_mem) + 63L) & ~63L);
